package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "azure-iot-toolkit",
-  "displayName": "Azure IoT Hub Toolkit",
-  "description": "Interact with Azure IoT Hub, IoT Device Management, IoT Edge Management, IoT Hub Device Simulation, IoT Hub Code Generation and IoT Hub Device Provisioning Service (DPS)",
+  "displayName": "Azure IoT Hub",
+  "description": "This extension is now a part of Azure IoT Tools extension pack. We highly recommend installing Azure IoT Tools to get full capabilities for Azure IoT development.",
```

Comment:
Also keep the previous content in description?

Reply:
Agree, we need keep the previous description to let user know what the extension help.